src/js/App/RootApp.js

```diff
@@ -14,6 +14,9 @@ import isEqual from 'lodash/isEqual';
 import { onToggle } from '../redux/actions';
 import Routes from './Routes';
 import useOuiaTags from '../utils/useOuiaTags';
+import { QuickStartDrawer, QuickStartContext, useValuesForQuickStartContext, useLocalStorage, QuickStartCatalogPage } from '@patternfly/quickstarts';
+import '@patternfly/quickstarts/dist/quickstarts.css';
```

Comment:
Can you import these in the `chrome.scss` file please?

Reply:
Yes, also I am currently cleaning up the global styles. PR will be on hold until that is fixed